install/kubernetes/helm/subcharts/pilot/values.yaml

```diff
@@ -21,3 +21,8 @@ env:
 cpu:
   targetAverageUtilization: 80
 nodeSelector: {}
+# SDS related options
+sdsOptions:
+  # The path to the SDS k8s token.
+  # If non-empty, this path is used to fetch the SDS k8s token.
```

Comment:
probably better rephrase something like if non-empty and useNormalJwt is set, envoy will fetch token from this path(instead of default k8s sa jwt path `/var/run/secrets/kubernetes.io/serviceaccount/token` ) and pass to sds server

Reply:
Done.